### PR TITLE
Adding taskgraph conda-forge recipe

### PR DESCRIPTION
**Pull request by phargogh**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by phargogh** [comment]
I'm willing to maintain this conda-forge package.

**Comment by phargogh** [comment]
@conda-forge/help-python would you be willing to take a look at this?  Thanks in advance!

File: recipes/taskgraph/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: d94f63039711607a059465f76a93b9c2dc4d68105978e1e1501a9a6dd1119043
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (synapticarbors) [inline recipes/taskgraph/meta.yaml:13]: Can this be `noarch: python`?

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Comment (phargogh) [reply recipes/taskgraph/meta.yaml]: Yep!  Patched in 0ce72f129

File: recipes/taskgraph/meta.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ source:
 
 build:
     number: 0
-    script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (synapticarbors) [inline recipes/taskgraph/meta.yaml]: You want to leave the `script` line that you deleted. Sorry for the confusion is I was unclear in my original comment.

Comment (phargogh) [reply recipes/taskgraph/meta.yaml]: Whoops!  Thanks for clarifying that ... this is my first `noarch: python` package.  Patched in b75f78197

**Pull request merged**